.drone.yml

```diff
@@ -10,7 +10,7 @@ pipeline:
   # build on pull requests
 
   build:
-    image: alpine:3.6
+    image: node:8.8-alpine
     environment:
       - BASE_URL="//blog.sourced.tech/"
```

Comment:
Set `ALLOW_UNPUBLISHED` to something also here or, better, set a default in `Makefile` and overwrite it in `.drone.yml` only when necessary.